modules/js/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ set(bindings_cpp "${CMAKE_CURRENT_BINARY_DIR}/bindings.cpp")
 
 set(scripts_hdr_parser "${CMAKE_CURRENT_SOURCE_DIR}/../python/src2/hdr_parser.py")
 
+set(JS_HELPER "${CMAKE_CURRENT_SOURCE_DIR}/src/helper.js")
```

Comment:
@pancx , please change the file name from `helper.js` to `helpers.js`, as it contains several helper functions. Thanks!